src/layout/header/Menu.tsx

```diff
@@ -82,59 +81,65 @@ const Menu = ({ menuButtonLabel, handleLogout }: MenuProps) => {
           vertical: 'bottom',
           horizontal: 'left',
         }}>
-        {user?.isCurator && (
-          <StyledAdminMenu>
-            <StyledNormalText>{t('profile:roles.curator')}</StyledNormalText>
+        {user?.isCurator && [
+          <StyledHeaderTypography key="1">{t('profile:roles.curator')}</StyledHeaderTypography>,
+          <MenuItem
+            key="menu-my-worklist-button"
+            data-testid="menu-my-worklist-button"
```

Comment:
alle testid-er skulle egentlig vært i enums, så kunne vi bare importert de både her og i testene. Hadde blitt enklere for @Eiriknil å vedlikeholde også. men gidder ikke å ta det nå